app/src/docs/v1.api-spec.yaml

```diff
@@ -408,7 +428,7 @@ paths:
                 $ref: '#/components/schemas/Error'
   /forms/{formId}/export/fields:
     post:
-      summary: Export submissions for a form with the ability to select fields to export if CSV is selected. Use only in frontend (from an interface)
+      summary: Export submissions for a form with the ability to select fields to export if CSV is selected.
```

Comment:
Since this can be called with the API Key, it's not really for only frontend use.